pulsar/producer_partition.go

```diff
@@ -752,6 +755,14 @@ func (p *partitionProducer) updateMetadataSeqID(mm *pb.MessageMetadata, msg *Pro
 	}
 }
 
+func (p *partitionProducer) updateSingleMessageMetadataSeqID(smm *pb.SingleMessageMetadata, msg *ProducerMessage) {
```

Comment:
It seems pointless to abstract this method

Reply:
I know that `updateSingleMessageMetadataSeqID` is not reused by other functions, I asctracted it as a method is in the purpose of that making the code is easy to maintain, we can know that we update the meta seqID with a same manner both for `MetadataSeqID` and `SingleMessageMetadataSeqID` obvoiusly.

Reply:
Thanks for your explain, I agree.